server/src/main/java/org/opensearch/repositories/blobstore/BlobStoreRepository.java

```diff
@@ -3753,27 +3753,42 @@ public void snapshotRemoteStoreIndexShard(
         String shardStateIdentifier,
         IndexShardSnapshotStatus snapshotStatus,
         long primaryTerm,
+        long commitGeneration,
         long startTime,
+        Map<String, Long> indexFilesToFileLengthMap,
         ActionListener<String> listener
     ) {
```

Comment:
Let's also override the other `snapshotRemoteStoreIndexShard ()` method which will call this method by passing indexFilesToFileLengthMap as null.